pyjobshop/solve.py

```diff
@@ -63,12 +66,15 @@ def solve(
         )
 
         cpoptimizer = CPOptimizerSolver(data)
-        return cpoptimizer.solve(
+        result = cpoptimizer.solve(
             time_limit,
             display,
             num_workers,
             initial_solution,
             **kwargs,
         )
-    else:
-        raise ValueError(f"Unknown solver choice: {solver}.")
+
+    if display:
+        print(result)
```

Comment:
The display of the result is already done by the solver. For example, ortools reports:

```
CpSolverResponse summary:
status: OPTIMAL
objective: 60
best_bound: 60
integers: 22
booleans: 110
conflicts: 0
branches: 0
propagations: 0
integer_propagations: 0
restarts: 0
lp_iterations: 0
walltime: 0.309544
usertime: 0.309544
deterministic_time: 0.853162
gap_integral: 1.24885
solution_fingerprint: 0x1b17f1b9c2766b98
```

So is this really necessary?

Reply:
Yeah good point!